README.md

```diff
@@ -11,7 +11,7 @@ The associated preprint can be found at:  https://doi.org/10.1101/2023.04.18.537
 A conda release will be coming soon. In the meantime, please install manually.
 
 ### Manual installation
-YACHT requires Python 3 or higher. We recommend using a virtual environment (such as conda) to install YACHT. To create a virtual environment, run:
+YACHT requires Python 3 or higher. We recommend using a virtual environment (such as conda) to install YACHT. Before you begin the installation, you must also install [Rust](https://rustup.rs/). To create a virtual environment, run::
```

Comment:
Is Rust really required to run YACHT itself? Rust is a requirement of Sourmash, but don't we cover that by saying "you need to have sourmash installed"?

Reply:
Yes, you are right,  will close merge request.